MericCelik/assignment3/Dictionary.java

```diff
@@ -0,0 +1,80 @@
+import java.util.HashSet;
+
+/**@author AycaMericCelik
+ * */
+
+public class Dictionary {
+    private String[] words;
```

Comment:
How about using HashSets instead of String[] for both words and prefixes? Then the lookup times are much faster (you could then just do words.contains(word) and prefixes.contains(prefix) instead of writing your own 'contains' method). :) You also don't need to worry about duplicates then.

Reply:
You are right! Corrected :smile: